__tests__/app.test.js

```diff
@@ -54,10 +54,57 @@ describe("GET API/reviews", () => {
             votes: expect.any(Number),
             designer: expect.any(String),
             comment_count: expect.any(Number),
+            category: expect.any(String),
           });
         });
 
         expect(body.reviews).toBeSorted("created_at", { descending: true });
       });
   });
 });
+
+describe("GET API/reviews/:review_id", () => {
+  test("should return 200 status & 1 review object with correct corresponding ID", () => {
+    const reviewId = 1;
+    return request(app)
+      .get(`/api/reviews/${reviewId}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          review: {
```

Comment:
can we test the data types  incase the data changes.
eg. using expect.objectContaining or .toMatchObject
we can test the review_id to equal 1 as this will never change